src/core/tables.js

```diff
@@ -0,0 +1,163 @@
+// @ts-check
+// Module core/tables
+// Handles tables in the document.
+// Generates a Table of Tables wherever there is a #tot element.
```

Comment:
Might be better to be more explicit `#table-of-tables`.
(`#tof` was probably a mistake; `#toc` is okay)

Reply:
Actually, it's a list 😄 
So, `#list-of-tables` or `#tables-list`.